libraries/app/application.cpp

```diff
@@ -351,6 +365,9 @@ namespace detail {
          }
          _chain_db->show_free_memory( true );
 
+         _next_rebroadcast = _chain_db->head_block_num() + 3 + rand() % 4;
```

Comment:
I'd keep this code identical with line 243. It's a candidate for refactoring after a couple more uses and keeping its expression consistent should help other developers recognize that later.